ietf/meeting/tests_views.py

```diff
@@ -7636,14 +7636,14 @@ def test_proceedings_overview(self, mock):
         response = self.client.get(url)
         self.assertContains(response, 'The Internet Engineering Task Force')
 
-    def test_proceedings_progress_report(self):
+    def test_proceedings_activity_report(self):
         make_meeting_test_data()
```

Comment:
Get out of the habit of using this, and build the data you need with Factories instead.